applications/vanilla/controllers/api/DiscussionsApiController.php

```diff
@@ -125,6 +125,11 @@ protected function fullSchema() {
             'insertUser?' => $this->getUserFragmentSchema(),
             'bookmarked:b' => 'Whether or no the discussion is bookmarked by the current user.',
             'announce:b' => 'Whether or not the discussion has been announced (pinned).',
```

Comment:
Shouldn't we remove announce?

Reply:
This'll be nuked as part of #5937. Right now, it's needed when using PUT announce.